### DISCUSSION TITLE
Can't Install when a private nuget feed is configured

### DISCUSSION THREAD

**Discussion by cfindlayvcs** [stryker-mutator/stryker-net#1569] [Q&A] [answered]
When I try to install, I get this:

![image](https://user-images.githubusercontent.com/73087164/118567299-e5129d80-b7b8-11eb-89fa-90a2e400c242.png)

It also doesn't tell me what the current version is so I can't specify the version

**Comment by rouke-broersma** [comment, marked as answer]
Hi @cfindlayvcs 

It looks like you are not authenticated with your private nuget feed on the command line:

![image](https://user-images.githubusercontent.com/3595932/118603265-fa38fd80-b7b3-11eb-91d6-5261d09430cc.png)

You could try following these instructions to authenticate to your feed in the command line: https://docs.microsoft.com/en-us/azure/devops/artifacts/nuget/dotnet-exe?view=azure-devops

So run `dotnet tool install -g dotnet-stryker --interactive`

Or you can specify 'ignore failed sources':

`dotnet tool install -g dotnet-stryker --ignore-failed-sources`

Or you can temporarily disable the unauthenticated package source: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-disable-source

`dotnet nuget disable source <source_name>`

**Reply by cfindlayvcs** [reply]
Oh wow, thanks for picking that up. I think I probably need to remove it from my sources. It's odd that the dotnet tools depends on my config of nuget feeds.

**Reply by rouke-broersma** [reply]
> Oh wow, thanks for picking that up. I think I probably need to remove it from my sources. It's odd that the dotnet tools depends on my config of nuget feeds.

Dotnet tools install from nuget, if you have some private tools you can put those in your own feed and it works!
